Domain/Domain.csproj

```diff
@@ -4,4 +4,13 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="3.1.9" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Entity Interfaces" />
```

Comment:
Remember - never use white space for naming folders, you can run into issues, and that issues will troubleshoot difficultly

Reply:
Got it. Resolved.